server/app/assets/javascript/route.coffee

```diff
@@ -87,7 +93,7 @@ vueConf = (id) ->
       @dest = route.dest
       @setHash()
     modalURL: (route) ->
-      base = "/entire/sta/route_fleet/#{@area}/#{@info}/#{route.dep}/#{route.dest}"
+      base = if @is1st then jsRoutes.controllers.ViewSta.routeFleet1st(@area, @info, route.dep, route.dest).url else jsRoutes.controllers.ViewSta.routeFleet2nd(@area, @info, route.dep, route.dest).url
```

Comment:
__[CoffeeLint]__ Line exceeds maximum allowed length Length is 200, max is 120 ([link](https://sider.review/gh/repos/16994149/pulls/434#issue-4040466))

<sub>posted by Sider</sub>